docs/dev/analysis/features/text/pictures-in-runs.rst

```diff
@@ -0,0 +1,105 @@
+Shapes in Runs
```

Comment:
There is an existing set of pages for inline shapes: http://python-docx.readthedocs.io/en/latest/dev/analysis/features/shapes/shapes-inline.html (this link is the middle one of three pages)

This should just be an elaboration of the one I linked to just above. Most of the bits like schema excerpts and so on are the same and don't need to be repeated.